### ISSUE TITLE
Sanitization bypass using HTML Entities

### ISSUE THREAD

**Issue by venil7** [chjj/marked#808]
see for details:
https://www.versioneye.com/Node.JS/marked/0.3.6

**Comment by Feder1co5oave** [comment]
That vulnerability report is wrong: as I commented on that page, the provided example does not result in code injection. That particular vulnerability was fixed in fd0d1a2.

Test case to show this:
```
$ marked --sanitize
[URL](javascript&#x3Awindow;alert&#40;1&#41;)
^D
<p></p>
```

**Comment by matt-** [comment]
That page correctly says: Patched versions: >=0.3.6  even though its reporting the issue for 0.3.6. 

Closing issue.
